### PR TITLE
Rename Redis to Server or Valkey in log of cluster_legacy.c

### PR DESCRIPTION
**Pull request by 9bany**
https://github.com/valkey-io/valkey/issues/207

### CODE REVIEW

**Review by Shivshankar-Reddy — APPROVED** [review]

File: src/cluster_legacy.c [hunk 1]
```diff
@@ -1011,10 +1011,10 @@ void clusterInit(void) {
      * us from trying to use a too-high cluster port number. */
     int port = defaultClientPort();
     if (!server.cluster_port && port > (65535-CLUSTER_PORT_INCR)) {
-        serverLog(LL_WARNING, "Redis port number too high. "
+        serverLog(LL_WARNING, "Valkey port number too high. "
```
Comment (hwware) [inline src/cluster_legacy.c:1014]: I am struggling with this: Valkey or Server @valkey-io/core-team

Comment (paulmchen) [reply src/cluster_legacy.c:1014]: Suggest we use "Server port number" instead.

Comment (zuiderkwast) [reply src/cluster_legacy.c:1014]: The question is whether we need redis compatibility in log entries. We've discussed adding a compatibility switch (compile time or runtime config). No decision yet though whether log entries should be part of that.

**Review by zuiderkwast — CHANGES_REQUESTED** [review]
Don't merge. See #207.

**Comment by 9bany** [comment]
merge to https://github.com/valkey-io/valkey/pull/226

**Pull request closed without merging**